app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,8 @@
+class QuotesController < ApplicationController 
+  def index
+    @keyword = params[:query].capitalize
+    quote_data = QuoteFacade.quotes_by_keyword(@keyword)
+    @quote_count = quote_data[:total_quotes]
+    @quotes = quote_data[:quotes]
+  end
```

Comment:
This looks great! I like how you're sending back two things from your facade method.